### ISSUE TITLE
[Suggestion, low priority] Provide "statistics" and information about the whole glimmer suite in some way, or perhaps several ways

### ISSUE THREAD

**Issue by rubyFeedback** [AndyObtiva/glimmer#25]
Hey there Andy,

Before I describe this issue report/request, let me explain how I came to it.

In one of your recent videos, I think it was "Parking", I saw one of the drop down
listings mention several games (or widgets) on the left hand side. I actually was
surprised to see that many widgets, so I am not quite up-to-date. :)

I assume it was for glimmer SWT primarily so I guess it may not work equally
well across all bindings; or at the least I think SWT is where you put in a LOT
of your time. A while ago I suggested kind of something like a "store" where 
users could "discover the glimmer world". E. g. a bit like the Microsoft store
or any other store; my idea was less about monetization per se (although 
this could be a side goal), but more about "discoverability" - e. g. if a user 
wants to know if this or that exists. Perhaps even via search functionality
+ tags, if at one point you have a tons of different ready-made widgets 
already. But actually, while I think a store-like app may be nice, ultimately I
think the more important part is to find what a user may want to find here
(in this case, me - so my suggestions are also mostly for selfish reasons
too :D ).

Would it be possible to perhaps have some kind of "cheat sheet"? Like if
you remember zenspider, the quickref he maintains: https://www.zenspider.com/ruby/quickref.html

Something similar to glimmer. Perhaps a table layout or something.

Something that is condensed, though, like for instance:

"Available Games:
game1
game2
game3"

And so forth. And then to display the status among different glimmer suites, 
e. g. "supported in SWT, TK" and so forth and a marked checkbox. Or 
some other way.

You already provide a LOT of information which is nice, but perhaps there
could be a central "overview" site too (and people able to find it; sometimes
you may not find what you are looking for).

This page could also show some statistics if you would like to. But this
is secondary; my primary rationale for this request is that people can 
quickly determine what works where. If the glimmer suite were a webpage
like rubygems.org, then a sub-API such as /stats or /statistics or 
/overview or something like that may be nice.

If such a detailed overview already exists then perhaps it could be
mentioned more prominently. Right now I am not sure on which
toolkits the games or customized widgets, including Parking, would
work. Would be neat if we can obtain that information quickly, ideally
via a webpage. This could also help users before they download other
applications, such as windows users before they download jruby and
play with it (or SWT perhaps).

Anyway, this is just a suggestion, please feel free to proceed as 
always - my aim is not to draw away too much from your time. Providing
information is super-important, but sometimes it seems hard at the 
least to me to be able to answer simple questions such as how 
strong the support is on some of the toollkits and what widgets may
be available.

**Comment by AndyObtiva** [comment]
You could create this information and submit a Pull Request when you're ready.

Otherwise, I am closing this issue because I don't personally see it as useful since people can easily discover applications by visiting the application links in each project, which I provided to you before in a previous issue. If you are unsure if an app is available in toolkit xyz, then simply visit toolkit xyz's webpage on GitHub, and you will discover if the app is in its samples/examples. It's that simple! I'm not sure what's so complicated about that.

Also, the way you're thinking about things is all wrong. If you see an app in Glimmer DSL for SWT, you should only focus on learning it, not distract yourself with other toolkits. Only after you've mastered it could you wonder if you could rebuild in other toolkits, and once you've mastered its code, you should do the rebuilding in other toolkits yourself, which will force you to learn them, instead of asking me to build the example in multiple toolkits. Afterwards, you can submit a pull request for me to merge your newly implemented app/sample into whatever project it was added to, and we both win that way. You win by learning through practice, and I win by getting a contribution for my project.

That's how open-source collaboration efforts work. Not by just asking all the time without contributing much yourself.
